testsuite/gnat2goto/tests/arrays_range/arrays_range.adb

```diff
@@ -5,7 +5,8 @@ procedure Arrays_Range is
       subtype My_Index is Integer range Arr'Range;
       An_Index : My_Index := 1;
    begin
-      pragma Assert(An_Index = 1);
+      An_Index := An_Index + 1; -- to invoke the range check
```

Comment:
Can you add a case that fails the range check (both for lower and upper bounds), just to satisfy my paranoia if nothing else?

Reply:
Done.